packages/duc/build.sh

```diff
@@ -0,0 +1,11 @@
+TERMUX_PKG_HOMEPAGE=https://www.stunnel.org/
```

Comment:
Forgot to add your home page URL?

Reply:
Yeah, having fun with copy&pasting, sorry for that